boards/cc2650stk/include/periph_conf.h

```diff
@@ -69,13 +69,20 @@ static const timer_conf_t timer_config[] = {
  * The used CC26x0 CPU only supports a single UART device, so all we need to
  * configure are the RX and TX pins.
  *
- * Optionally we can enable hardware flow control, by setting UART_HW_FLOW_CTRL
- * to 1 and defining pins for UART_CTS_PIN and UART_RTS_PIN.
+ * Optionally we can enable hardware flow control, by setting flow_control
+ * to 1 and defining pins for cts_pin and rts_pin.
  * @{
  */
-#define UART_NUMOF          (1)
-#define UART_RX_PIN         (28)
-#define UART_TX_PIN         (29)
+
+static const uart_conf_t uart_config[] = {
+ {
+     .tx_pin = 29,
+     .rx_pin = 28,
+     .flow_control = 0,
+     .intn = UART0_IRQN
+ }
```

Comment:
Yea, C++ does not support leaving gaps or changing the order of the struct members when initializing them like that.
This results in those `sorry, unimplemented: non-trivial designated initializers not supported` errors.

Annoying, I know - I thought C++ was supposed to be more advanced than C?